restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/http/CallsEndpointTest.java

```diff
@@ -82,11 +84,52 @@ public void getCallsList() {
         assertTrue(totalSize == 448);
     }
 
+    @Test
+    public void getCallsListUsingSorting() {
+        // Provide both sort field and direction
+        // Provide ascending sorting and verify that the first row is indeed the earliest one
+        JsonObject response1 = RestcommCallsTool.getInstance().getCalls(deploymentUrl.toString(),
+                adminAccountSid, adminAuthToken, 0, 10, "date_created:asc", true);
+        assertTrue(((JsonObject)response1.get("calls").getAsJsonArray().get(0)).get("date_created").getAsString().equals("Fri, 5 Jul 2013 22:15:53 +0300"));
```

Comment:
i know this is well spread in the testsuite, but could you please switch this into proper assertEquals, then when assertion fails the message its much clearer. Please amend all occurences in this test class please.

Reply:
Sure, makes sense

Reply:
@jaimecasero fixed the issues in IT, and created PR to see if the CDR related ones are fixed. On the last failure on 'org.restcomm.connect.testsuite.sms.push.SmsPushNotificationServerTest.testSmsEndpointMessage' I think it's unrelated to my changes. Can you verify?

Reply:
right SmsPushNotificationServerTest is unstable test on master, we need to mark it unstable, but dont worry about this now... its not related to this PR

Reply:
Cool